xbmc/filesystem/File.cpp

```diff
@@ -205,22 +205,22 @@ bool CFile::CURLCreate(const std::string &url)
   return true;
 }
 
-bool CFile::CURLAddOption(XFILE::CURLOPTIONTYPE type, const char* name, const char * value)
+bool CFile::CURLAddOption(XFILE::CURLOptionType type, const char* name, const char * value)
```

Comment:
`XFILE` was removed everywhere else in the filesystem subdir, might as well remove it here too for consistency.

Reply:
done.